tests/StarterTheme/specs/customer/my-account.js

```diff
@@ -45,5 +45,32 @@ describe('My account page', function () {
       ;
     });
 
+    it('should display credit splips link', function () {
+      return browser
+        .isExisting('.page-content a#order-slips-link')
```

Comment:
Yes, that's right but it easier to read back.
That way, we know we are looking at a link element.